examples/src/app.js

```diff
@@ -49,6 +49,7 @@ var StatesField = React.createClass({
 		});
 	},
 	focusStateSelect: function() {
+		console.log(this.refs);
```

Comment:
I'll remove this one. But the other ones are there to explain that the selector is capable of throwing events. This is file the demo app of the lib.